app/src/gecko/java/org/mozilla/focus/web/GeckoViewPrompt.java

```diff
@@ -466,7 +486,8 @@ private static int parseColor(final String value, final int def) {
         }
     }
 
-    public void promptForColor(final GeckoSession session, final String title,
+    @Override
+    public void onColorPrompt(final GeckoSession session, final String title,
```

Comment:
\[PMD\] INFO: The method 'onColorPrompt' has a Standard Cyclomatic Complexity of 10\.